index.php

```diff
@@ -10,29 +10,33 @@
     <title>Guitar Collection</title>
     <link rel="stylesheet" type="text/css" href="normalize-8-0-1.css">
     <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Catamaran&display=swap"
+          rel="stylesheet">
 </head>
 <body>
+<div class="container">
 <div>
     <h1>Guitar Collection</h1>
-    <p>These are the things in my collection</p>
+    <p>These are the things in my collection:</p>
 </div>
-<div>
+<div class="guitarsContainer">
     <div class="row header">
-        <div class="item-detail">#</div>
-        <div class="item-detail">Image</div>
-        <div class="item-detail">Guitar</div>
-        <div class="item-detail">Year</div>
-        <div class="item-detail">Type</div>
-        <div class="item-detail">Country</div>
-        <div class="item-detail">Value</div>
-        <div class="item-detail">Date Acquired</div>
+        <div class="item-detail column1">#</div>
+        <div class="item-detail column2">Image</div>
+        <div class="item-detail column3">Guitar</div>
+        <div class="item-detail column4">Year</div>
+        <div class="item-detail column5">Type</div>
+        <div class="item-detail column6">Country</div>
+        <div class="item-detail column7">Value</div>
```

Comment:
I suspect you don't want to output the values purely as integers, maybe concatenate a £ and a '.00' on the end